src/createConnectedRouter.js

```diff
@@ -1,33 +1,25 @@
-import { connect } from 'react-redux';
+import React from 'react';
+import { shallowEqual, useSelector, useStore } from 'react-redux';
```

Comment:
would it make sense to put in some assert in the non-connected router that warns if it detects the existence of redux?

i can think of two ways where this might go wrong, theoretically
1. something somehow imports something else that eventually imports redux, therefore accidentally importing redux into the package. Just knowing not to connect them and being careful doesn't seem like a great solution in an open source system
2. (much more likely) someone imports the index file and therefore pulls in redux, and then uses the non-connected router

i suppose there's the case where you're using redux but _don't_ want the connected router, though I'm not sure why you would want that...

Reply:
I think what i want to do for a breaking change is remove this entirely, and make the `createFarceRouter` the only factory, if you want to integrate with redux pass a store in, otherwise we will use vanilla redux as an implementation detail